java/src/jmri/jmrix/loconet/LncvDevicesManager.java

```diff
@@ -46,12 +46,15 @@
  */
 
 public class LncvDevicesManager extends PropertyChangeSupport
-        implements LocoNetListener {
+        implements LocoNetListener, jmri.Disposable {
     private final LocoNetSystemConnectionMemo memo;
     @GuardedBy("this")
     private final LncvDevices lncvDevices;
 
-    public LncvDevicesManager(LocoNetSystemConnectionMemo memo) {
+    // constant for thread name, with memo prefix appended.
+    static final String ROSTER_THREAD_NAME = "rosterMatchingListLncvDM";
+
+    public LncvDevicesManager(@javax.annotation.Nonnull LocoNetSystemConnectionMemo memo) {
```

Comment:
I personally prefer in this case Nonnull, instead of the complete package, since that makes the code easier to read.

Reply:
Agreed, I was mulling on these.
A single import seemed waseful, however when constructors are involed it's a different matter.
In future, constructors are the exception👍